test/addAction.test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai');
+const { Add } = require('../lib/actions/addClass');
+
+describe('Add Action', ()=> {
+    it('@asyncResolves : should return an add object containing a response', (done) => {
+        let add = new Add();
+        add.execute('label url', {links: {}})
+
+        const resolvingPromise = new Promise( (resolve) => {
```

Comment:
I think I see what you were going for here. Definitely thinking in the right direction. Per @nrambeck's feedback, we want the `execute` method to return a promise rather than create, resolve, and handle it in the test which is not doing much for our "system under test" (aka the Action class).

Reply:
i seeeee i seeee.. gotchaaa B) 
I'm on et.